Added new forecasts (01-24-2021)

sakethsundar opened the pull request:
## Description

Added new forecasts (01-24-2021)


> _If this section does not apply to you, please delete it in your PR_  

If you are **adding new forecasts** to an existing model, please include the following details:- 
- Team name: MIT-CriticalData
- Model name that is being updated:  MIT_CritData-GBCF
---

## Checklist

- [X ] Specify a proper PR title with your team name.
- [X ] All validation checks ran successfully on your branch. Instructions to run the tests locally is present [here](https://github.com/reichlab/covid19-forecast-hub/wiki/Running-Checks-Locally).

eycramer commented:
Hi @sakethsundar, 

Can you please delete the file from 1/17 from this PR if it was already submitted last week? 

If you are making an update to the file from last week, can you please comment explaining what the update is? 

Thanks!

sakethsundar commented:
let me delete 01/17

sakethsundar commented:
Hi @eycramer

Reverted yesterday commit. Added only 01-24 forecasts back. Please let me know whether our forecasts for this week will be included in CDC state-level forecasts and viz dashboard.

Thanks
Saketh
